packages/react-mutation-mapper/src/store/DefaultMutationMapperStore.ts

```diff
@@ -724,9 +790,9 @@ class DefaultMutationMapperStore implements MutationMapperStore {
         {
             await: () => [this.cancerHotspotsData],
             invoke: async () =>
-                this.ptmData.result
```

Comment:
this seems like a bug, we shouldn't be waiting for ptm data to process cancer hotspots data.